frontend/src/metabase/browse/components/CollectionBreadcrumbsWithTooltip.tsx

```diff
@@ -85,52 +86,62 @@ export const CollectionBreadcrumbsWithTooltip = ({
-                  style={{ alignItems: "center" }}
-                  w="auto"
-                  display="flex"
+        <CollectionLink to={Urls.collection(collection)}>
+          <Flex
+            align="center"
+            w="100%"
+            lh="1"
+            style={{ flexFlow: "row nowrap" }}
+          >
+            <CollectionsIcon
+              name="folder"
+              // Stopping propagation so that the parent <tr>'s onclick won't fire
+              onClick={(e: React.MouseEvent) => e.stopPropagation()}
+            />
+            {shownCollections.map((collection, index) => {
+              const key = `collection${collection.id}`;
+              return (
+                <BreadcrumbGroup
+                  spacing={0}
+                  key={key}
+                  // Stopping propagation so that the parent <tr>'s onclick won't fire
+                  onClick={(e: React.MouseEvent) => e.stopPropagation()}
                 >
-                  {index === 0 && !justOneShown && (
-                    <Ellipsis
-                      ref={initialEllipsisRef}
-                      includeSep={false}
-                      className="initial-ellipsis"
-                    />
-                  )}
-                  {index > 0 && ellipsifyPath && <Ellipsis />}
-                  <Breadcrumb
-                    href={Urls.collection(collection)}
-                    className={classNames("breadcrumb", `for-index-${index}`, {
-                      "sole-breadcrumb": collections.length === 1,
-                    })}
-                    ref={(el: HTMLAnchorElement) => ref.current.set(key, el)}
-                    maw={maxWidths[index]}
-                    key={collection.id}
+                  {index > 0 && <PathSeparator />}
+                  <CollectionBreadcrumbsWrapper
+                    containerName={containerName}
+                    style={{ alignItems: "center" }}
+                    w="auto"
+                    display="flex"
                   >
-                    {getCollectionName(collection)}
-                  </Breadcrumb>
-                </CollectionBreadcrumbsWrapper>
-              </BreadcrumbGroup>
-            );
-          })}
-        </Flex>
+                    {index === 0 && !justOneShown && (
+                      <Ellipsis
+                        ref={initialEllipsisRef}
+                        includeSep={false}
+                        className="initial-ellipsis"
+                      />
+                    )}
+                    {index > 0 && ellipsifyPath && <Ellipsis />}
+                    <Breadcrumb
+                      maxWidth={maxWidths[index]}
+                      className={classNames(
+                        "breadcrumb",
+                        `for-index-${index}`,
+                        {
+                          "sole-breadcrumb": collections.length === 1,
+                        },
+                      )}
```

Comment:
we should not be setting class names likes this. this will break our embedding library